src/DSharp.Compiler.Tests/Source/Type/Reflection/Metadata.txt

```diff
@@ -1,17 +1,18 @@
 ss.registerMetadataImporter(function(){(function(ss) {
   "use strict"
   var Void = null;
+  var Type = function(){};
   var module = ss.modules['test'];
   module.MyClass.$members = [
-    {MemberType: 16,Name: 'Other', Type: module.MyClass_$1},
-    {MemberType: 16,Name: 'Getter', Type: Number},
-    {MemberType: 8,Name: 'method', Type: Void},
-    {MemberType: 8,Name: 'method2', Type: Object},
-    {MemberType: 8,Name: 'method3', Type: ss.IEnumerable},
-    {MemberType: 8,Name: 'method4', Type: ss.IList_$1},
-    {MemberType: 8,Name: '_assert', Type: Void},
-    {MemberType: 8,Name: 'get_item', Type: Object},
-    {MemberType: 8,Name: 'set_item', Type: Object}];
+    {MemberType: 16,Name: 'Other', Type: eval("try{module.MyClass_$1;}catch{}"},
```

Comment:
missing `)` in the end

Reply:
semicolon can be removed it can be `eval("try{module.MyClass_$1}catch{})"`

Reply:
fixed